shell/platform/windows/flutter_windows_engine.h

```diff
@@ -203,6 +204,19 @@ class FlutterWindowsEngine {
   // Returns the native accessibility node with the given id.
   gfx::NativeViewAccessible GetNativeAccessibleFromId(AccessibilityNodeId id);
 
+  // Register a root isolate create callback.
+  //
+  // The root isolate create callback is invoked at creation of the root Dart
+  // isolate in the app. This may be used to be notified that execution of the
+  // main Dart entrypoint is about to begin, and is used by test infrastructure
+  // to register a native function resolver that can resolve functions marked as
+  // native in the Dart code.
+  //
+  // This must be called before calling |Run|.
+  void SetRootIsolateCreateCallback(fml::closure callback) {
```

Comment:
Should this be a const reference?

```suggestion
  void SetRootIsolateCreateCallback(const fml::closure& callback) {
```

Reply:
Nice catch! Yes it should. Done!